src/game/game.cpp

```diff
@@ -10735,3 +10666,15 @@ Title Game::getTitleByName(const std::string &name) {
 	}
 	return {};
 }
+
+std::map<uint8_t, std::string> Game::getBlessingNames() {
+	return m_blessingNames;
+}
+
+std::unordered_map<uint16_t, std::string> Game::getHirelingSkills() {
+	return m_hirelingSkills;
+}
+
+std::unordered_map<uint16_t, std::string> Game::getHirelingOutfits() {
+	return m_hirelingOutfits;
+}
```

Comment:
Isn't better to return the reference to avoid unnecessary copying of these maps? Could set as const method because the method does not update the variable and if the map shouldn't be update could return as const as well.

```suggestion
std::map<uint8_t, std::string> &Game::getBlessingNames() const {
	return m_blessingNames;
}

std::unordered_map<uint16_t, std::string> &Game::getHirelingSkills() const {
	return m_hirelingSkills;
}

std::unordered_map<uint16_t, std::string> &Game::getHirelingOutfits() const {
	return m_hirelingOutfits;
}
```

Reply:
If it you want to pass the map as const.

```suggestion
const std::map<uint8_t, std::string> &Game::getBlessingNames() const {
	return m_blessingNames;
}

const std::unordered_map<uint16_t, std::string> &Game::getHirelingSkills() const {
	return m_hirelingSkills;
}

const std::unordered_map<uint16_t, std::string> &Game::getHirelingOutfits() const {
	return m_hirelingOutfits;
}
```